tsconfig.json

```diff
@@ -2,7 +2,6 @@
   "compilerOptions": {
     "target": "es2015",
     "module": "nodenext",
-    "moduleResolution": "nodenext",
```

Comment:
forces ts-jest to use `"moduleResolution": "nodenext",`
see https://www.typescriptlang.org/docs/handbook/modules/reference.html#implied-and-enforced-options